packages/ui5-task-stringreplacer/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "ui5-task-stringreplacer",
-    "version": "0.5.5",
+    "version": "0.6.0",
```

Comment:
You don't need to update the version - this happens via conventional commits. A `feat` change will create a minor version upgrade.